src/components/PromotionModal.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+
+import close from '../assets/close.svg';
+
+import styles from './PromotionModal.module.css';
+import FocusTrap from './FocusTrap';
+
+const PromotionModal = () => {
+  const [isOpen, setIsOpen] = useState(true);
+
+  const closeModal = (e: React.MouseEvent<HTMLButtonElement | HTMLDivElement>) => {
+    e.preventDefault();
+    setIsOpen(false);
+  };
+
+  if (!isOpen) {
+    return null;
+  }
+
+  return (
+    <FocusTrap>
+      <div className={styles.modal} aria-modal="true" role="dialog">
+        <div className={styles.modalBackdrop} onClick={closeModal}></div>
+        <div className={styles.modalContainer}>
+          <div className={styles.modalContent}>
+            <h1
+              id="modal-title"
+              className={`${styles.modalTitle} heading-2-text`}
+              aria-hidden="true"
+            >
+              여행할 땐 A11Y AIRLINE 앱
+            </h1>
+            <p
+              id="modal-description"
+              className={`${styles.modalDescription} body-text`}
+              aria-hidden="true"
+            >
+              체크인, 탑승권 저장, 수하물 알림까지
+              <br />- 앱으로 더욱 편하게 여행하세요!
+            </p>
+            <div
+              role="document"
+              aria-labelledby="modal-title"
+              aria-describedby="modal-title modal-description"
+            >
+              <button className={`${styles.modalActionButton} button-text`} autoFocus>
+                앱에서 열기
+              </button>
+            </div>
+            <button
+              className={`${styles.modalCloseButton} heading-2-text`}
+              onClick={closeModal}
+              aria-label="창 닫기"
+            >
+              <img src={close} alt="" />
+            </button>
```

Comment:
저도 이 부분에 대해서 고민을 많이 하다가, W3C Tutorial을 참고해 1번으로 결정했어요!

[W3 - Tutorial/Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
> This image is used as a border in the page design and has a purely decorative purpose.
> ```html
> <img src="topinfo_bg.png" alt="">
> ```


다만 아래 문서들을 참고하니, `role="none"` 혹은 `role="presentation"`도 추가해 주는 것이 좋다는 생각이 드네요!
role="presentation" 속성을 추가했습니다 😁

[Image has non-empty accessible name](https://www.w3.org/WAI/standards-guidelines/act/rules/23a2a8/)
> **Expectation**
Each target element has an [accessible name](https://www.w3.org/WAI/standards-guidelines/act/rules/23a2a8/#accessible-name) that is not empty (""), or has a [semantic role](https://www.w3.org/WAI/standards-guidelines/act/rules/23a2a8/#semantic-role) of none or presentation.

[W3C - ARIA in HTML](https://www.w3.org/TR/html-aria/)
> If the img has an empty alt="" attribute and no aria-label or aria-labelledby attributes to provide it an accessible name: [No role](https://www.w3.org/TR/html-aria/#dfn-no-role) other than the role=[none](https://www.w3.org/TR/html-aria/#index-aria-none) or [presentation](https://www.w3.org/TR/html-aria/#index-aria-presentation) roles, which are NOT RECOMMENDED.

[wai-aria 1.1](https://www.w3.org/TR/wai-aria-1.1/)
> For any element with an explicit or inherited role of presentation, user agents MUST apply an inherited role of presentation to all host-language-specific labeling elements for the presentational element. For example, a table element with a role of presentation will have the implicit native semantics of its caption element removed, because the caption is merely a label for the presentational table.

Reply:
꼼꼼한 레퍼런스 소개에 감사드립니다. 저도 알려주신 내용 참고해서 이후에 접근성 관련 작업할 때 반영해야겠어요 👍